R/rstudio.R

```diff
@@ -39,33 +39,17 @@ use_rstudio <- function(line_ending = c("posix", "windows")) {
 #' be found in the blog post [Project-oriented
 #' workflow](https://www.tidyverse.org/blog/2017/12/workflow-vs-script/).
 #'
-#' Only `use_blank_slate("project")` is automated so far, since RStudio
-#' currently only supports modification of user-level or global options via the
-#' user interface.
-#'
 #' @inheritParams edit
 #'
 #' @export
 use_blank_slate <- function(scope = c("user", "project")) {
   scope <- match.arg(scope)
 
-  if (scope == "user") { # nocov start
-    ui_todo(
-      "To start ALL RStudio sessions with a blank slate, \\
-      you must set this interactively, for now."
-    )
-    ui_todo(
-      "In {ui_field('Global Options > General')}, \\
-      do NOT check {ui_field('Restore .RData into workspace at startup')}."
-    )
-    ui_todo(
-      "In {ui_field('Global Options > General')}, \\
-      set {ui_field('Save workspace to .RData on exit')} to {ui_value('Never')}."
-    )
-    ui_todo(
-      "Call {ui_code('use_blank_slate(\"project\")')} to opt in to the \\
-      blank slate workflow for a specific project."
-    )
+  if (scope == "user") {
+    use_rstudio_config(list(
+      save_workspace = "never",
```

Comment:
Do you think we should announce anything about what we're doing? I guess it's a simple function and this is the only reason to call it.

Reply:
`use_rstudio_config()` should do the notification